module-ama/src/main/kotlin/org/quiltmc/community/cozy/modules/ama/data/MemoryAmaData.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package org.quiltmc.community.cozy.modules.ama.data
+
+import dev.kord.common.entity.Snowflake
+
+public class MemoryAmaData : AmaData {
+	private val ama: MutableList<AmaConfig> = mutableListOf()
```

Comment:
Nitpick: might it be reasonable to instead use a map here?

Reply:
Why?

Reply:
Since configs are only referenced by guild ID, it could give a slight speedup. It also fixes a potential bug: currently any changes caused by running `setConfig` will be entirely ignored as only the first matching item is chosen

Reply:
I think it would be a `guildId` to `AmaConfig` map